scripts/deploy-helm.sh

```diff
@@ -421,7 +421,7 @@ info ""
 info "- 4. Create dynatrace-gcp-function IAM role(s)."
 if [[ $DEPLOYMENT_TYPE == logs ]] || [[ $DEPLOYMENT_TYPE == all ]]; then
   debug "Creating or updating GCP IAM role with gcp_iam_roles/dynatrace-gcp-function-logs-role.yaml"
-  if [[ $(gcloud iam roles list --filter="name:$ROLE_NAME.logs" --project="$GCP_PROJECT" --format="value(name)") ]]; then
+  if [[ $(gcloud iam roles list --filter="name ~ $ROLE_NAME.logs" --project="$GCP_PROJECT" --format="value(name)") ]]; then
```

Comment:
To avoid future warnings:
WARNING: --filter : operator evaluation is changing for consistency across Google APIs

Ref: https://cloud.google.com/sdk/gcloud/reference/topic/filters